### DISCUSSION TITLE
Is there a way to configure a @dbt_asset with asset check?

### DISCUSSION THREAD

**Discussion by marcilj** [dagster-io/dagster#21701] [Q&A] [answered]
I know we can activate the dbt translator to configure asset check from DBT to be loaded for DBT assets.

But can we create asset checks in Dagster for @dbt_asset in python?

**Comment by johannkm** [comment, marked as answer]
Yes, you'll just need to pass in the asset key. E.g. here's a check defined on the `customers` model of dbt's jaffle shop example https://github.com/dagster-io/dagster/commit/4c52b63442163362b7c0670c91092c460b94fdeb#diff-62fe1b5411c63c04fce52f5114a594c5e5333a42fad2f683701b8209d1d34704R372-R374

**Reply by marcilj** [reply]
Doesn't seem to work on 1.7.3.

I've just revert back to 1.6.1 and I can see the asset check
![image](https://github.com/dagster-io/dagster/assets/89793010/a1664dc2-1633-47e0-a388-751520c2960f)

But I have nothing when trying this with 1.7.3

**Reply by johannkm** [reply]
Could you test with 1.7.4? That included a bug fix that might be behind this

**Reply by marcilj** [reply]
Thanks @johannkm That work.

Damm I was so confused why this wasn't working as expected. Thank you for reaching out quickly.
